test/unit_test/passes/onnx/test_perf_tuning.py

```diff
@@ -11,7 +12,14 @@
 from olive.passes.onnx import OrtPerfTuning
 
 
-@pytest.mark.parametrize("config", [{"input_names": ["input"], "input_shapes": [[1, 1]]}, {}])
+@pytest.mark.parametrize(
+    "config",
+    [
+        {"input_names": ["input"], "input_shapes": [[1, 1]]},
+        {"providers_list": ["CPUExecutionProvider", "CUDAExecutionProvider"], "device": "gpu"},
```

Comment:
need assert the EP used by perf tuning is the list besides checking none.

Reply:
Fixed in another PR.https://github.com/microsoft/Olive/pull/700